versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md

```diff
@@ -12,7 +12,6 @@ import TableTextSmall from "./assets/react-components/TableTextSmall";
 import userTaskMigrationDecisionHelperForm from "./assets/forms/userTaskMigrationDecisionHelperForm.js";
 import "./assets/css/condensedTable.module.css";
 import styles from "./assets/css/cleanImages.module.css";
-import APIArchitectureImg from './assets/img/api-architecture.png';
```

Comment:
@christinaausley this line removal is causing your build error. This image is referenced in the doc, and can't be removed.

Reply:
THANK YOU!